Elysian-Outpost/Assets/Scripts/SizedTerrainGeneration.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Threading.Tasks;
+using Unity.Mathematics;
+using UnityEngine;
+using UnityEngine.Serialization;
+using VoxelMaster;
+
+public class SizedTerrainGeneration : MonoBehaviour
+{
+    /// <summary>
+    /// Size of the terrain in chunks.
+    /// </summary>
+    [SerializeField] private int3 terrainSize = new int3(16, 16, 5);
```

Comment:
usually I prefer having the two sizes, since its not a big size taken... It would allow lately the ability to have rectangular maps (for some mods if needed)

Reply:
Alright then, so let's leave it like that!